app/uk/gov/hmrc/softdrinksindustrylevystub/Store.scala

```diff
@@ -42,21 +42,19 @@ object Store {
     def generate(pred: Subscription => Boolean) =
       genSubscription.retryUntil(pred).seeded(sdil)
 
-    val seeded = (sdil.init.last, sdil.last) match {
-      case (_, '0')   => None
-      case ('1', '1') => generate(_.activity.isSmallProducer).map(_.copy(warehouseSites = Nil))
-      case (_, '1')   => generate(_.activity.isSmallProducer)
-      case ('2', '2') => generate(_.activity.isLarge).map(_.copy(warehouseSites = Nil))
-      case ('3', '2') => generate(_.activity.isLargeNoImports).map(_.copy(warehouseSites = Nil))
-      case (_, '2')   => generate(_.activity.isLarge)
-      case ('3', '3') => generate(_.activity.isLargeImportCopacker)
-      case (_, '3')   => generate(_.activity.isImporter)
-      case (_, '4')   => generate(_.activity.isContractPacker)
-      case (_, '5')   => generate(_.activity.isVoluntaryRegistration)
-      case (_, '6') =>
```

Comment:
7, 8, 9 are free to work with. Perhaps change values used in ATs if affecting this functionality